python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/_tracer.py

```diff
@@ -140,6 +145,10 @@ def _start_trace(self, run: Run) -> None:
             context=parent_context,
             start_time=start_time_utc_nano,
         )
+
+        if run.run_type.lower() == "chain" and span:
+            span.set_attribute(IS_CHAIN_SPAN, True)
```

Comment:
1. What are the benefits of modifying the span, versus keeping this information external to the span, e.g. in a tuple or a custom object?
2. What if in the future users want a different `run_type`? How adaptable is this approach: do we need to reserve additional attribute keywords for this purpose?